engine/labels/labels_test.go

```diff
@@ -360,6 +360,23 @@ func Test_ruleSelector_Matches(t *testing.T) {
 			},
 			want: true,
 		},
+		{
+			name: "simple wildcard value",
+			expr: "(konveyor.io/source=technology-usage)",
+			ruleLabels: []string{
+				"konveyor.io/source",
+			},
+			want: true,
+		},
+		{
+			name: "wildcard value expression",
+			expr: "(konveyor.io/target && konveyor.io/source=test)",
```

Comment:
Hmm, this reads a little confusing to me

This looks like it says, `konveyor.io/target` exists at all, and the `konveyor.io/source` has to exist, and be equal to `test`

But then the label below just has a `source`, with no value, which I guess we're basically treating like `source=*`. Does this mean there's no way to exclude a ruleset that doesn't provide a value for a label?  I guess we'd have that same behavior whether we treated `*` or `""` as the wildcard value so maybe it doesn't matter. 

Feel like we'll need a lot of docs around this behavior though because I find it pretty unintuitive just looking at the selector vs labels

Reply:
@fabianvf to exclude, you will need to do a `!konveyor.io/source`  or to specifically exclude something with `test` value `!konveyor.io/source=target`

Reply:
yeah it makes sense, just think we'll need to doc it well. It's more intuitive to have wildcards in the selectors than the labels, that's probably what's throwing me off.